### PR TITLE
update version and contact info in setup.py

### PR DESCRIPTION
**Pull request by KatiRG**
## What this PR accomplishes
Updates [setup.py](https://github.com/ongov/ckanext-ontario_theme/blob/ckan_2.9_ver/setup.py) to include:
- correct python version (3.8)
- current contact info

Updates CKAN version to 2.9.x in [README](https://github.com/ongov/ckanext-ontario_theme/blob/ckan_2.9_ver/README.rst)

## What needs review
- confirm that contact info is correct

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -26,8 +26,8 @@
     url='https://github.com/boykoc/ckanext-ontario_theme',
```
Comment (aimalkhan) [inline setup.py:26]: this should be updated to `https://github.com/ongov/ckanext-ontario_theme`

Comment (aimalkhan) [inline setup.py:26]: `version='0.0.1'` on line 20 should be corrected to the versioning scheme we are using. I would like a comment on this from @bianca 
current release v1.0 according to tags (https://github.com/ongov/ckanext-ontario_theme/releases)

**Review by aimalkhan — COMMENTED** [review]
Looks good, but i wanted a couple of changes. I want the requested theme version changes to be reviewed by @bianca

File: setup.py (hunk 1 above)
Comment (KatiRG) [reply setup.py:26]: sorry! missed that

Comment (aimalkhan) [reply setup.py]: update to 2.0.0, please.

**Pull request merged**